it_hardware/data/ir_model_fields.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="new_date_lot_serial_x_warranty_date" model="ir.model.fields">
+        <field name="name">x_warranty_date</field>
+        <field name="model_id" ref="stock.model_stock_lot"/>
+        <field name="ttype">date</field>
+        <field name="field_description">Warranty Date</field>
+    </record>
+    <record id="new_date_helpdesk_ti_x_warranty_date" model="ir.model.fields">
+        <field name="name">x_warranty_date</field>
+        <field name="model_id" ref="helpdesk.model_helpdesk_ticket"/>
+        <field name="ttype">date</field>
+        <field name="field_description">Warranty Date</field>
+        <field name="related">lot_id.x_warranty_date</field>
+    </record>
+    <record id="new_decimal_lead_opp_x_no_of_quantity" model="ir.model.fields">
+        <field name="name">x_no_of_quantity</field>
+        <field name="model_id" ref="crm.model_crm_lead"/>
+        <field name="ttype">float</field>
+        <field name="field_description">No Of Quantity</field>
+    </record>
+    <record id="new_integer_product_x_warranty_date" model="ir.model.fields">
+        <field name="name">x_warranty_date</field>
+        <field name="model_id" ref="product.model_product_template"/>
+        <field name="ttype">integer</field>
+        <field name="field_description">Warranty Days</field>
+    </record>
```

Comment:
This doesn't seem to be correct. Warranty is on the lot, not the product. This should either be removed or used as default for the lot